control-plane/csi-driver/controller/src/client.rs

```diff
@@ -283,11 +283,12 @@ impl IoEngineApiClient {
         volume_id: &uuid::Uuid,
         node: Option<&str>,
         protocol: VolumeShareProtocol,
+        republish: bool,
```

Comment:
I'm not sure that we should be exposing this via rest to prevent misuse..

Reply:
Yeah, I also had this thought but I added it so that we can have the bdd-tests, as the bdd tests use the rest api.